src/main/java/com/exasol/releasedroid/github/GithubAPIAdapter.java

```diff
@@ -155,7 +160,50 @@ private void sendGitHubRequest(final URI uri, final String json) throws GitHubEx
     private void validateResponse(final HttpResponse<String> response) throws GitHubException {
         if ((response.statusCode() < HttpURLConnection.HTTP_OK)
                 || (response.statusCode() >= HttpURLConnection.HTTP_MULT_CHOICE)) {
-            throw new GitHubException("F-RR-GH-6: An HTTP request failed. " + response.body());
+            throw new GitHubException("F-RR-GH-6: An executing workflow HTTP request failed. " + response.body());
+        }
+    }
+
+    private String getWorkflowConclusion(final String repositoryFullName, final String workflowName)
+            throws GitHubException {
+        while (true) {
+            waitOneMinute();
+            final URI uri = createUriFromString(
+                    GITHUB_API_ENTRY_URL + repositoryFullName + "/actions/workflows/" + workflowName + "/runs");
```

Comment:
Extract and abstract.

Reply:
I don't know what else I can abstract here, but I extracted a common string part